docs/src/pages/style/icons.md

```diff
@@ -12,7 +13,7 @@ Google has provided a set of [Material icons](https://material.io/icons/) that f
 
 Material-UI provides two components to render system icons: `Icon` for rendering font icons, and `SvgIcon` for rendering SVG paths.
 
-### Font Icons
+## Font Icons
```

Comment:
Font Icons is subsection of System icons:

"Material-UI provides two components to render system icons: `Icon` for rendering font icons, and `SvgIcon` for rendering SVG paths."